Shoko.Server/Renamer/RenameFileHelper.cs

```diff
@@ -116,6 +116,7 @@ public static (ImportFolder, string) GetDestination(SVR_VideoLocal_Place place,
                     if (args.Cancel) return (null, null);
                     // if no path was specified, then defer
                     if (string.IsNullOrEmpty(destPath) || destFolder == null) continue;
+                    destPath = destPath.Replace(Path.AltDirectorySeparatorChar, Path.DirectorySeparatorChar);
```

Comment:
Problem with this solution, Linux lists `/` as both values, which makes it a pointless replace.;

```suggestion
                    if (Path.AltDirectorySeparatorChar != Path.DirectorySeparatorChar) {
                        destPath = destPath.Replace(Path.AltDirectorySeparatorChar, Path.DirectorySeparatorChar);
                    }
```